spk/gerrit/Makefile

```diff
@@ -0,0 +1,54 @@
+SPK_NAME = gerrit
+SPK_VERS = 2.11.4
+SPK_REV = 1
+SPK_ICON = src/gerrit.png
+DSM_UI_DIR = app
+
+DEPENDS = cross/busybox
+SPK_DEPENDS = "JavaManager:git"
```

Comment:
Please add restrictions for models/arches that don't support JavaManager. Use `UNSUPPORTED_ARCHS`, like [this](https://github.com/SynoCommunity/spksrc/blob/master/spk/sonarr/Makefile#L8)